package.json

```diff
@@ -31,7 +31,8 @@
     "web": "cd apps/next && yarn next",
     "build": "turbo run build",
     "web-clear-cache": "cd apps/next && rm -rf .next/cache && yarn next",
-    "build-watch": "turbo run build-watch"
+    "build-watch": "turbo run build-watch",
+    "build-app": "turbo run build-app"
```

Comment:
[Question] What does this do? I presume it runs the same command in the expo app, but how?

Reply:
this will run the "build-app" command (specified in the turbo.json as its a turbo command) in any package it can find that contains that command - so we can use this to build expo, next and tv apps. I set the command in the turbo.json to also do any necessary builds before running the command to "build-app"